src/Controllers/UserController.js

```diff
@@ -90,7 +90,7 @@ export class UserController extends AdaptableController {
       )
       .then(results => {
         if (results.length != 1) {
-          throw undefined;
```

Comment:
did a little git blame sleuthing on this one.

git show 7e868b2dc

note the comment that used to be there: 
"      
        // Trying to verify email when not enabled
       // TODO: Better error here.
"

Reply:
I didn't even notice that.